src/autocomplete/PeopleAutocomplete.js

```diff
@@ -32,6 +33,17 @@ class PeopleAutocomplete extends PureComponent<Props> {
     const { users, onAutocomplete } = this.props;
     const user = users.find(x => x.email === email);
     if (user) {
+      const idx = binarySearch(users, elem => {
+        if (elem.full_name === user.full_name && elem.user_id === user.user_id) {
+          return 1;
+        }
+        return elem.full_name.toLowerCase().localeCompare(user.full_name.toLowerCase());
+      });
```

Comment:
This won't work: the given comparator doesn't describe a consistent ordering.

Consider `user = { full_name: "John Doe", user_id: 4, ... }`, and a `users` array matching the following:
```javascript
users =~ [
  { full_name: "A", user_id: 0 }, // -1
  { full_name: "B", user_id: 1 }, // -1
  { full_name: "C", user_id: 2 }, // -1
  { full_name: "D", user_id: 3 }, // -1
  { full_name: "John Doe", user_id: 4 }, // 1
  { full_name: "John Doe", user_id: 5 }, // 0
  { full_name: "X", user_id: 6 }, // 1
  { full_name: "Y", user_id: 7 }, // 1
  { full_name: "Z", user_id: 8 }, // 1
];
```
(Trailing comments show the expected return value of the comparator for each element.)

`binary_search` will check `users[4]` first, which is the user in question; but the comparator will return `1`, so it will limit further searches to the range `[0, 3]`, and therefore never find `{ full_name: "John Doe", user_id: 5 }`.

Reply:
Fixed now.